sections/crystal.zsh

```diff
@@ -0,0 +1,37 @@
+#
+# Crystal
+#
+# Crystal is a programming that aimes to be "fast as C, slick as Ruby."
+# Link: https://www.crystal-lang.org
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_CRYSTAL_SHOW="${SPACESHIP_CRYSTAL_SHOW=true}"
+SPACESHIP_CRYSTAL_PREFIX="${SPACESHIP_CRYSTAL_PREFIX="$SPACESHIP_PROMPT_DEFAULT_PREFIX"}"
+SPACESHIP_CRYSTAL_SUFFIX="${SPACESHIP_CRYSTAL_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_CRYSTAL_SYMBOL="${SPACESHIP_CRYSTAL_SYMBOL="💎 "}"
```

Comment:
Symbol looks identical to symbol used in `ruby` section. 

https://github.com/denysdovhan/spaceship-prompt/blob/fa10285731af1bfeedf776d3af01426f3bd7a688/sections/ruby.zsh#L14